pyiron_gui/project/project_browser.py

```diff
@@ -140,40 +495,34 @@ def __getattr__(self, item):
     def clear_output(self, *args, **kwargs):
         clear_button = kwargs.pop('clear_button', False)
         if clear_button:
-            self.buttons = widgets.HBox()
+            self.header = widgets.HBox()
```

Comment:
`self.header` gets refreshed like this a few times, but it's a bit difficult to follow. I suppose it has something to do with the `refresh` method?  Might be worth to change it to `self.refresh(header=True)`, but this is strictly extra credit.

Reply:
I just realize, I actually do not use the `header` of the `DisplayOutputGUI` any more. This was a place for persisting a button connected to a specific object as long as the `DisplayOutputGUI` did not get an entirely new object to display. However, now I shifted all of such things to the `ObjectWidget` instances. I will remove the `header` and the complicated logic to persist it in specific cases altogether.

Reply:
Also I try to maintain the `refresh` method in all cases to be without any argument such that one may call `.refresh()` on a GUI element to just refresh it.